vega/utils.py

```diff
@@ -48,7 +48,7 @@ def parse_object_column_type(val):
         if isinstance(val, np.ndarray):
             return val.tolist()
         elif isinstance(val, datetime.date):
-            return "{dt:%Y-%m-%d}".format(dt=val)
+            return val.isoformat()
```

Comment:
Unfortunately, a partial iso format like ``"2019-01-01"`` is not sufficient to ensure that Javascript date parsing is consistent with the expectations of pandas interfacing to Vega-Lite. You need the full iso format, like ``'2019-01-01T00:00:00'``.

Reply:
Thank you for clearing this up for me i have done more than a couple hacks dealing with this.